### PR TITLE
Prep for CI #2 

### PR DESCRIPTION
**Pull request by tramstheman**
**Overview:** 

- Finished prepping django environments for future CI
- Added documentation for standing up local database and django environment. 
- Added django admin for data models. 
- Showcasing a normalized version of the existing data model (showcased in "election_data" django app). I recommend we move to a data model that more closely resembles what is being showcased in "election-data". The old data model was left for compatibility. 
- Migration files added to project

**Details**
Django best practices suggest that each Django app (python module) should exist with one purpose in mind. For this reason the data model for the election data was put into it's own Django app and the API can exist in its own Django app. For the purpose of compatibility and collaboration, the old data model is left intact within the api Django app. This data model is not normalized. The data models presented in "election_data" illustrate how the data could be normalized. The data models themselves should also be re-examined at some point as almost every field is a text field and none of the fields are indexed (at least in the Django representation of the data model). This may warrant a discussion with other groups. 

A requirements.txt was also added for the purpose of making work out of Pycharm or other IDE's easier as they can open the project from the "congressionaldata" directory. 

A dev and local settings file were added to the settings. Documentation was also added in the form of a README to show how to stand up a local Django run end-to-end. The documentation caters mainly to mac users.

### CODE REVIEW

File: django/congressionaldata/congressionaldata/settings/dev.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from congressionaldata.settings.common import *
+import os
+import urllib.parse
+
+# Read DB URI From Environment Variable
+CD_DWH = os.getenv('CD_DWH')
+
+PARSED_DBURI = urllib.parse.urlparse(CD_DWH)
+
+ALLOWED_HOSTS = []
```
Comment (JessDF) [inline django/congressionaldata/congressionaldata/settings/dev.py]: I pulled your branch and tried to run python manage.py runserver and recieved this error:
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.

According to this [stackoverflow](https://stackoverflow.com/questions/24857158/commanderror-you-must-set-settings-allowed-hosts-if-debug-is-false): The host header (or X-Forwarded-Host if USE_X_FORWARDED_HOST is enabled) should match one of the values in ALLOWED_HOSTS. 

I see the next line has CORS_ORIGIN_WHITELIST declared though with the correct hosts

Comment (JessDF) [inline django/congressionaldata/congressionaldata/settings/dev.py]: I pulled your branch and tried to run python manage.py runserver and recieved this error:
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.

According to this [stackoverflow](https://stackoverflow.com/questions/24857158/commanderror-you-must-set-settings-allowed-hosts-if-debug-is-false): The host header (or X-Forwarded-Host if USE_X_FORWARDED_HOST is enabled) should match one of the values in ALLOWED_HOSTS. 

I see the next line has CORS_ORIGIN_WHITELIST declared though with the correct hosts #

File: django/congressionaldata/README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+Then check the running version.
+```bash
+postgres -V
+```
+
+##### Everyone!
+
+```bash
+psql postgres # You may need to run this with sudo
```
Comment (JessDF) [inline django/congressionaldata/README.md]: [Step 5 on this doc](https://github.com/sfbrigade/datasci-congressional-data/blob/master/onboarding/02_development_environment.md#for-windowsubuntu) says to run `psql -p 5432 -h localhost -U postgres` instead.

I also receive the following errors when running either command on the local server ( I pulled your master branch)

image

Comment (tramstheman) [reply django/congressionaldata/README.md]: We worked together on slack and resolved this issue. I am going to update the docs to include windows/ubuntu solutions.

**Pull request merged**